lib/logger.js

```diff
@@ -0,0 +1,21 @@
+import readline from 'readline'
```

Comment:
Would it be worth adding some functions here (error, success, warning, instruction) that simply extend the 'log' function but with chalk colours. Then I think chalk would only be imported here and the logger calls would be more informative to the next developer (logger.error being more obvious than logger.log(chalk.red(...)))

Just an idea as an enhancement. Happy for it to be added to the TODO.md instead of in here if it is decided to be worthwhile

Reply:
genius